src/main/java/com/websafe/ccmsbe/controller/PrivacyRegulationController.java

```diff
@@ -0,0 +1,53 @@
+package com.websafe.ccmsbe.controller;
+
+import com.websafe.ccmsbe.entity.PrivacyRegulation;
+import com.websafe.ccmsbe.service.PrivacyRegulationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@CrossOrigin
+@RequestMapping("api/v1/website-management")
+public class PrivacyRegulationController {
+    private final PrivacyRegulationService privacyRegulationService;
+    @Autowired
+    public PrivacyRegulationController(PrivacyRegulationService privacyRegulationService) {
```

Comment:
When writing a function in Java, leave a blank line before the function body. The same goes for the classes.

Reply:
Fixed 425c298